### PR TITLE
make the logging of failed social auth optional

### PR DESCRIPTION
**Pull request by MounirMesselmeni**
make the logging of failed authentication requests optional using a settings value

### CODE REVIEW

File: rest_social_auth/views.py [hunk 1]
```diff
@@ -175,7 +176,8 @@ def get_provider_name(self, input_data):
 
     def respond_error(self, error):
         if isinstance(error, Exception):
-            self.log_exception(error)
+            if not isinstance(error, AuthException) or LOG_AUTH_EXCEPTIONS:
```
Comment (st4lk) [inline rest_social_auth/views.py]: Am I right, that this is needed not to spam sentry for example?
Also, why not include `AuthException`

**Comment by st4lk** [comment]
Hi Mounir!
Thanks for PR. Have some question ^

File: rest_social_auth/views.py (hunk 1 above)
Comment (MounirMesselmeni) [inline rest_social_auth/views.py]: Yes, that's exactly why. What do you mean by including AuthException?

Comment (st4lk) [inline rest_social_auth/views.py]: Ah, nevermind, I treated this condition wrongly at first sight.

**Pull request merged**